modules/get-started/pages/release-notes/redpanda.adoc

```diff
@@ -18,6 +18,27 @@ The Redpanda Admin API now includes new health probes to help you ensure safe br
 * xref:manage:cluster-maintenance/rolling-restart.adoc[]
 * xref:upgrade:rolling-upgrade.adoc[]
 
+== HTTP Proxy authentication changes
+
+[IMPORTANT]
+====
+**Breaking change**: Starting in Redpanda 25.2, ephemeral credentials for HTTP Proxy are removed. If your HTTP Proxy API listeners use `authentication_method: none`, you must configure explicit SASL credentials for HTTP Proxy to authenticate with the Kafka API.
+====
+
+HTTP Proxy previously used automatically generated ephemeral credentials to authenticate with the Kafka API when the HTTP Proxy listeners had `authentication_method: none`. These ephemeral credentials are now removed to improve security and simplify the authentication model.
+
+Without explicit SASL credentials configured, HTTP Proxy attempts unauthenticated connections to the Kafka API.
+
+Redpanda Data recommends configuring authentication for HTTP Proxy to ensure secure communication with the Kafka API.
```

Comment:
```suggestion
Redpanda Data recommends configuring authentication for HTTP Proxy to ensure secure communication with the Kafka API. Without explicit SASL credentials configured, HTTP Proxy attempts unauthenticated connections to the Kafka API.


```

Reply:
> users can still simply go without configuring authN connections to the Kafka API

I think the answer to this is yes - if you don't configure auth the Kafka API and the Kafka API has auth on requests will fail. If you enable auth on the HTTP proxy, then we will forward those credentials to the Kafka API for authentication. This last method is what we recommend - Kafka and HTTP auth should be enabled/disabled together, there should never be a mismatch